src/index.mjs

```diff
@@ -13,18 +15,23 @@ const logger = jsLogger.default({
   prettyPrint: env.get('LOG_PRETTY').default('false').asBool(),
 });
 
+const GEOSERVER_LOCAL_PORT = '8080'; // Default port for local GeoServer instance, hard coded in deployment.yaml containerPort
+const POLLING_INTERVAL = env.get('POLLING_INTERVAL').default(300).asIntPositive(); // Polling interval in milliseconds
 const GEOSERVER_BASE_URL = env.get('GEOSERVER_BASE_URL').default('http://localhost:8080/geoserver').asString();
+const GEOSERVER_LOCAL_BASE_URL = `http://localhost:${GEOSERVER_LOCAL_PORT}/geoserver`;
 
 const GEOSERVER_API_BASE_URL = env.get('GEOSERVER_API_BASE_URL').default('http://localhost:8081').asString();
 const CATALOG_MANAGER_SERVICE_URL = env.get('CATALOG_MANAGER_SERVICE_URL').default('http://localhost:8082').asString();
 
 const WORKSPACE_NAME = env.get('WORKSPACE_NAME').default('polygonParts').asString();
 const DATASTORE_NAME = env.get('DATASTORE_NAME').default('polygonParts').asString();
+const DATASTORE_PATH = env.get('DATASTORE_PATH').default('/data_dir/workspaces/polygonParts/polygonParts').asString();
```

Comment:
change the hardcoded 'polygonParts' ('workspace' & 'datastore') values to consumed by the existing envs:
WORKSPACE_NAME
DATASTORE_NAME